packages/vibrant-components/src/lib/ResourceList/ResourceItem.tsx

```diff
@@ -35,7 +35,8 @@ export const ResourceItem = withResourceItemVariation(
 
     const handleSelect = useCallback(() => {
       if (isControlled) {
-        onToggleSelect?.({ id, selected: selectable ? !isSelected : true });
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        onToggleSelect?.({ id: id as any, selected: selectable ? !isSelected : true });
```

Comment:
`ResourceItemProps` 타입을 수정하면 type safe 하기 때문에 아래와 같이 써도 됩니다!

```suggestion
        onToggleSelect?.({ id, selected: selectable ? !isSelected : true });
```